src/pages/grants/grantsPage/sections/CommunityVoting.tsx

```diff
@@ -51,150 +32,48 @@ export const CommunityVoting = ({
 
-        return (
-          <CardLayout p={2} key={project.id}>
-            <Box display="flex">
-              {
-                <Box mr={3} height={{ base: '90px', lg: '144px' }}>
-                  <Link
-                    to={projectLink}
-                    className={classNames(classes.image, isMobile ? classes.mobileImage : classes.desktopImage)}
-                  >
-                    <ImageWithReload
-                      objectFit="cover"
-                      borderRadius="7px"
-                      width={isMobile ? '90px' : '144px'}
-                      height={isMobile ? '90px' : '144px'}
-                      src={project.thumbnailImage || ''}
-                      alt="project thumbnail"
-                    />
-                  </Link>
-                </Box>
-              }
-              <Box pr={2} flexGrow={1}>
-                <Link to={projectLink}>
-                  <H3 fontSize="18px">{project.title}</H3>
-                </Link>
-                <Link to={projectLink}>
-                  <Text noOfLines={4} wordBreak="break-word">
-                    {project.shortDescription}
-                  </Text>
-                </Link>
-              </Box>
-              {!isMobile && (
-                <Box
-                  minWidth="166px"
-                  pr={4}
-                  display="flex"
-                  flexDirection="column"
-                  justifyContent="center"
-                  alignItems="center"
-                >
-                  {renderButton(project)}
-                  {(grantHasVoting || isClosed) && renderWidgetItem(funding, contributorsCount)}
-                </Box>
-              )}
-            </Box>
-            {contributors && contributors.length > 0 && (
-              <Box pl={2} filter="opacity(0.4)">
-                {contributors.map(
-                  (contributor) =>
-                    contributor && (
-                      <AvatarElement
-                        key={contributor.user?.id}
-                        width="28px"
-                        height="28px"
-                        wrapperProps={{
-                          display: 'inline-block',
-                          marginLeft: '-5px',
-                          marginTop: 2,
-                        }}
-                        avatarOnly
-                        borderRadius="50%"
-                        seed={contributor?.user?.id}
-                        user={contributor?.user}
-                      />
-                    ),
-                )}
-              </Box>
-            )}
-            {isMobile && (
-              <VStack w="full">
-                {(grantHasVoting || isClosed) && renderWidgetItem(funding, contributorsCount)}
-                {renderButton(project)}
-              </VStack>
-            )}
-          </CardLayout>
-        )
-      })}
-      {modalProps.isOpen && <ProjectFundingModal {...modalProps} />}
-    </CardLayout>
+        <AuthModal
```

Comment:
Note: login is not enough, the user has to be logged in with a social account (ie: all besides lnurl).

Reply:
Is this synonymous with a user being anonymous or not?Is it true to say that a none-anonymous (`!anonymous`) loggedin user can vote?

Reply:
No, anonymous would be logged out. I can add a field `hasSocialAccount` on the graphql User object to help with this.

Reply:
See: https://github.com/geyserfund/geyser-server/pull/427/commits/1005f82c4327ec98ebba56fc200722c78078aa76

Reply:
Yes you do that and I will add it to the auth context in the frontend so it can be reused